test/unit/views/EoriEnrolSuccessSpec.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.views
+
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.api.test.FakeRequest
+import play.api.test.Helpers.contentAsString
+import uk.gov.hmrc.eoricommoncomponent.frontend.models.Service
+import uk.gov.hmrc.eoricommoncomponent.frontend.views.ServiceName._
+import uk.gov.hmrc.eoricommoncomponent.frontend.views.html.eori_enrol_success
+import util.ViewSpec
+
+class EoriEnrolSuccessSpec extends ViewSpec {
+
+  implicit val request = withFakeCSRF(FakeRequest())
+
+  private val service = Service.ATaR
+  private val eori    = "GB234532132435"
+
+  private val view = app.injector.instanceOf[eori_enrol_success]
```

Comment:
Probably worth to start using Injector trait that we have in different projects, but here you have application in viewSpec, so it won't change anything. Let's keep it for now, but something to keep in mind